### PR TITLE
Html

### PR DESCRIPTION
**Pull request by blainehowerton**
I'm done with the HTML for now.  I added a controller called Home so the app has an index in the home directory.  I also placed a bootstrap template on the index with some content (which is mostly just there as a holder for now).  I have updated the database structure on this branch to match the master.  It is functioning.  Please review and let's merge the branch if you're ready.

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController
+
+	def index
+	end
+
```
Comment (Ninjef) [inline app/controllers/home_controller.rb]: Minor: I think it's considered good practice, and cleaner, to not have so much whitespace.

File: config/database.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ default: &default
   adapter: mysql2
   pool: 5
   timeout: 5000
-  # password: Welltok1
+  password: zzS)_kI_b9Ug
```
Comment (Ninjef) [inline config/database.yml]: Didn't mean to upload my password here. Your approach with using an ENV var in your last repo was correct, as that way your PW doesn't go online. I'd change this to an environment variable and maybe change your password, :-P

**Comment by Ninjef** [comment]
Very minor comments, otherwise :+1:

**Comment by blainehowerton** [comment]
Awesome feedback!  Thank you.

**Pull request merged**